src/converter.js

```diff
@@ -239,11 +233,8 @@ converter.toObject = function toObject(mtype) {
             if (field.resolvedType instanceof Enum) gen
         ("d%s=o.enums===String?%j:%j", prop, field.resolvedType.valuesById[field.typeDefault], field.typeDefault);
             else if (field.long) gen
-        ("if(util.Long){")
-            ("var n=new util.Long(%i,%i,%j)", field.typeDefault.low, field.typeDefault.high, field.typeDefault.unsigned)
-            ("d%s=o.longs===String?n.toString():o.longs===Number?n.toNumber():n", prop)
-        ("}else")
-            ("d%s=o.longs===String?%j:%i", prop, field.typeDefault.toString(), field.typeDefault.toNumber());
+            ("var n=new util.LongBits(%i,%i,%j)", field.typeDefault.low, field.typeDefault.high, field.typeDefault.unsigned)
```

Comment:
This code will define n for every long field, which cause failure on es6 build(`-w es6`).
You should check if n has declared in previous field.
Like this : https://github.com/seo-rii/protobuf.js/commit/1dac06f2521694c6cb49dead3e571dbb1933c07f#diff-444e086c8038faf674140e30c0597966b9a9102859bf04422d247118adedd08c